spec/lib/utils/account_setter_spec.rb

```diff
@@ -0,0 +1,128 @@
+require 'rails_helper'
+
+RSpec.describe AccountSetter do
+  subject(:account_setter) { described_class.new(emails) }
```

Comment:
**nit** one of the benefits of described class is it is not tied to a particular name, so if the actual class names changes you can just change the `RSpec.describe` to the new/changed class name and the tests still pass. In that context i would make the named subject something more generic like `instance`
```
subject(:instance) { described_class.new(emails) }
```

Reply:
:+1: I'll change that.